apps/daimo-mobile/src/view/screen/SettingsScreen.tsx

```diff
@@ -327,4 +387,10 @@ const styles = StyleSheet.create({
   kvKey: {
     width: 128,
   },
+  pendingDot: {
+    width: 16,
+    height: 16,
+    borderRadius: 16,
+    backgroundColor: color.yellow,
```

Comment:
can pull `<PendingDot />` into shared component, but also fine to leave for a future pr

Reply:
done